### PR TITLE
Fix setup.cfg to include packages

### PR DESCRIPTION
**Pull request by dwhswenson**
Fixes remaining issue for installing as discussed in #22.

Also moves the CI import testing so we're not in the repo directory, to ensure that we're actually testing the import of the installed package.

### CODE REVIEW

**Review by IAlibay — APPROVED** [review]
Either way this solution works so I'll approve, but please see below for comments.

File: .github/workflows/ci.yaml
Comment (dwhswenson) [reply .github/workflows/ci.yaml]: Yep -- I hadn't been aware of `python -I`; thanks! Checked that it does mean that `sys.path` no longer starts with `''`. Changed to do it that way.

File: setup.cfg
Comment (dwhswenson) [reply setup.cfg:29]: `findnamespace:` can be a little dangerous because you might have something that looks like a package but isn't (they give an example where a `tests` directory gets identified as a package). That's why I avoid it.

**Comment by IAlibay** [comment]
Note: I'm sorta going ahead with the philosophy that we have enough to do that on a lot of these non-API things, if there's at least one agreeing reviewer it's good to go - do correct me if you want to have final say on things @richardjgowers

**Pull request merged**
